File: python/prism_sdk/throughput_resource_discovery_workflow_fabric.py
"""Worldgen P05 AFA-worldgen-P05-F15 product wrapper."""
from .worldgen_resource_workflow_support import ResourceWorkflowRequest, ResourceWorkflowReceipt, manifest as _manifest, schedule as _run
FEATURE_ID="AFA-worldgen-P05-F15"
CONTRACT_VERSION="worldgen-throughput-resource-workflow/1.0"
def worldgen_throughput_resource_discovery_workflow_fabric_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="ResourceWorkflowRequest1@1", scale="prospective high-throughput", autonomy_tier="A2")
def schedule_worldgen_throughput_resource_discovery_workflow(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="prospective high-throughput", require_approval=True, require_federation=False)
__all__=["ResourceWorkflowRequest","ResourceWorkflowReceipt","worldgen_throughput_resource_discovery_workflow_fabric_manifest","schedule_worldgen_throughput_resource_discovery_workflow"]

File: crates/worldgen/src/throughput_resource_discovery_workflow_fabric.rs
//! Worldgen P05 AFA-worldgen-P05-F15 throughput workflow_fabric.
use super::resource_workflow_support::{self,ResourceWorkflowRequest,ResourceWorkflowReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P05-F15"; pub const CONTRACT_VERSION:&str="worldgen-throughput-resource-workflow/1.0";
pub fn worldgen_throughput_resource_discovery_workflow_fabric_manifest()->serde_json::Value{resource_workflow_support::manifest(FEATURE_ID,CONTRACT_VERSION,"ResourceWorkflowRequest1@1","prospective high-throughput","A2")}
pub fn schedule_worldgen_throughput_resource_discovery_workflow(request:&ResourceWorkflowRequest)->Result<ResourceWorkflowReceipt,resource_workflow_support::ResourceWorkflowError>{resource_workflow_support::schedule(request,FEATURE_ID,CONTRACT_VERSION,"prospective high-throughput",true,false)}
pub use resource_workflow_support::{ResourceWorkflowError,ResourceWorkflowReceipt as WorldgenthroughputResourceworkflowfabricReceipt,ResourceWorkflowRequest as WorldgenthroughputResourceworkflowfabricRequest};
